server/notification-providers/hostedsms.js

```diff
@@ -0,0 +1,48 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { UP, DOWN } = require("../../src/util");
```

Comment:
now down is unused 😉 
```suggestion
const { UP } = require("../../src/util");
```